stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/file/utils/Constants.java

```diff
@@ -33,4 +33,10 @@
     public static final String FILE_NOT_FOUND_ERROR = "FileNotFoundError";
     public static final String FILE_NOT_OPENED_ERROR = "FileNotOpenedError";
     public static final String IO_ERROR = "IOError";
+
+    public static final String ACCESS_MODE_READ = "R";
+    public static final String ACCESS_MODE_WRITE = "W";
+    public static final String ACCESS_MODE_READ_AND_WRITE = "RW";
+    public static final String ACCESS_MODE_APPEND = "A";
+    public static final String ACCESS_MODE_READ_AND_APPEND = "RA";
```

Comment:
Can have an enum instead of constants for this.

Reply:
These are used for an `equals()` check. So using an enum wouldn't be much difference since we'll have to take the string value of the enum anyways right?